ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -520,6 +520,8 @@ public static void closeCurrent() {
    */
   private Hive(HiveConf c, boolean doRegisterAllFns) throws HiveException {
     conf = c;
+    // turn off calcite rexnode normalization
+    System.setProperty("calcite.enable.rexnode.digest.normalize", "false");
```

Comment:
I think this should be set here? https://github.com/apache/hive/blob/master/pom.xml#L1512

Reply:
I'm worried that it would only change this setting for the scope of the tests - and not for the release itself.

Reply:
Maybe the best option in terms of code changes and to be sure that the property is going to be taken into account is to add a `saffron.properties` file in the classpath and put the property in it.